examples/vue/package.json

```diff
@@ -19,8 +19,8 @@
     "@vitejs/plugin-vue": "^1.2.5",
     "@vue/compiler-sfc": "^3.0.5",
     "rimraf": "^3.0.2",
-    "vite": "^2.4.2",
-    "vite-plugin-pages": "^0.18.0",
+    "vite": "^2.9.9",
```

Comment:
I saw that vite 3.0 was released, do you know if the Authenticator works with the new version smoothly?

Reply:
I just did a quick smoke test and it was still working. But, this might be a good chance to add Vite 3 to our canary in a future PR.